### PR TITLE
Various doc fixes

### LINKED ISSUE #3751: Formula (?) in Periodic_2_triangulation_2
## Issue Details

In the file: `Periodic_2_triangulation_2/doc/Periodic_2_triangulation_2/CGAL/Periodic_2_triangulation_2.h` there is 4 times the construct: `\pre $0 \\leqle i \\leqle 2$`, this probably has to be `\pre \f$0 \leq i \leq 2\f$` ( and this does work in a `\pre`, tested for 1.8.13) or `\pre \f$ i \in\{0,1,2\}\f$` as used with `periodic_point`.

### PR DESCRIPTION
**Pull request by sloriot**
Fixes  #3751, #3756

### CODE REVIEW

File: AABB_tree/include/CGAL/internal/AABB_tree/AABB_ray_intersection.h [hunk 1]
```diff
@@ -180,7 +180,14 @@ class AABB_ray_intersection {
...
     typedef FT result_type;
-    as_ray_param_visitor(const Ray* ray) : ray(ray) {}
+    as_ray_param_visitor(const Ray* ray)
+     : ray(ray), max_i(0)
+    {
+      typename AABB_traits::Geom_traits::Vector_3 v = ray->to_vector();
+      for (int i=1; i<3; ++i)
+        if( CGAL::abs(v[i]) > CGAL::abs(v[max_i]) )
+          max_i = i;
+    }
```
Comment (lrineau) [inline AABB_tree/include/CGAL/internal/AABB_tree/AABB_ray_intersection.h]: That does not look like a doc fix. Was it supposed to be in this PR?

Comment (sloriot) [reply AABB_tree/include/CGAL/internal/AABB_tree/AABB_ray_intersection.h]: An extra commit slipped in. Fixed.

**Comment by maxGimeno** [comment]
Fusion conflicts in Arrangement_on_surface_2/doc/Arrangement_on_surface_2/CGAL/Arr_polycurve_traits_2.h : 

<<<<<<< HEAD
/// \ingroup PkgArrangementOnSurface2Macros
/// @{
/*!
 * If the macro is set to one, then \f$x\f$-monotone curves are always
 * directed from left-to-right.
 */
#define CGAL_ALWAYS_LEFT_TO_RIGHT
/// @}

=======
>>>>>>> sloriot/CGAL-doc_fixes

I remove it from integration for now

**Comment by sloriot** [comment]
pushed in integration

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-4.14-Ic-151.shtml

**Pull request merged**